invitations/app_settings.py

```diff
@@ -73,4 +73,11 @@ def EMAIL_SUBJECT_PREFIX(self):
         """
         return self._setting("EMAIL_SUBJECT_PREFIX", None)
 
+    @property
+    def INVITATION_MODEL(self):
+        """
+        Subject-line prefix to use for email messages sent
+        """
+        return self._setting("INVITATION_MODEL", "invitations.Invitation")
```

Comment:
currently i used the format used by get_model method here https://github.com/django/django/blob/master/django/apps/registry.py#L179